hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ApiStatusEnricher.java

```diff
@@ -27,19 +29,49 @@
  */
 public class ApiStatusEnricher extends AbstractTraceEnricher {
 
+  public static String getStatusCode(Event event, Protocol protocol) {
```

Comment:
Can we also move such methods to the bottom of the class? Since, these are utility methods, we usually follow to keep them at the bottom of the class and have the actual logic at the top?

Reply:
Done